operations/firmware/FAS_Recipes.md

```diff
@@ -20,10 +20,42 @@ Refer to [FAS Filters](FAS_Filters.md) for more information on the content used
 
 > **IMPORTANT:** Before updating a CMM:
 >
-> - Make sure all slot and rectifier power is off.
-> - The `hms-discovery` job must also be stopped before updates and restarted after updates are complete.
->   - (`ncn-mw#`) Stop `hms-discovery` job: `kubectl -n services patch cronjobs hms-discovery -p '{"spec":{"suspend":true}}'`
->   - (`ncn-mw#`) Start `hms-discovery` job: `kubectl -n services patch cronjobs hms-discovery -p '{"spec":{"suspend":false}}'`
+> - The `hms-discovery` job must be stopped before updates are done and restarted after updates are complete.
+>   - Stop the `hms-discovery` job.
+>
+>     ```bash
+>     ncn-mw# kubectl -n services patch cronjobs hms-discovery -p '{"spec":{"suspend":true}}'
```

Comment:
Please make copy-paste friendly for release/1.3 and main.

https://github.com/Cray-HPE/docs-csm/blob/main/introduction/documentation_conventions.md#command-prompt-location